.github/workflows/docs.yml

```diff
@@ -43,3 +43,22 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           BRANCH: gh-pages
           FOLDER: docs/.vuepress/dist
+
+  Generate-Swagger-and-Deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+
+      - name: Build 🔧
+        run:
+          make proto-swagger-gen
+
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@3.7.1
```

Comment:
Can we add an `if` condition to only deploy if the branch is the `master`? I would avoid deploying the docs straight from the PRs